src/main/java/com/purbon/kafka/topology/model/TopicDescription.java

```diff
@@ -0,0 +1,44 @@
+package com.purbon.kafka.topology.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+import java.util.Objects;
+
+public class TopicDescription {
```

Comment:
Instead of having a generic placeholder, would it be nice to reuse the same model. Do you think that is possible?

Reply:
I tried to first go with the new class since I wanted to have a model which does not depend on hierarchical information (topology -> project -> topics)